syntax = "proto3";
package Jubo.JuLiao.IM.Wx.Proto; //命名空间约定
import "TransportMessage.proto";
message FriendTalkNoticeMessage {
    string WeChatId = 1; // 商家个人微信内部全局唯一识别码
    string FriendId = 3; // 好友微信内部全局唯一识别码
    EnumContentType ContentType = 5; // 发送的消息内容类型
    bytes Content = 6; // 内容 二进制流
    int64 MsgId = 7;  //服务端的主键id
    int64 msgSvrId = 8; // 消息唯一id
    string Ext = 9; // 扩展信息，图片视频{"length":10000, "hdlen":20000, "duration":30} 文件大小，原图大小，视频时长
    int64 CreateTime = 10; // 消息时间
    string NickName = 11; // 发送者昵称
}